compute_sdk/tests/unit/test_client.py

```diff
@@ -320,17 +333,24 @@ def __init__(self):
     assert type(excinfo.value) is SystemError
 
 
-def test_register_function(gcc):
+def test_register_function(gcc, serde):
     gcc._compute_web_client = mock.MagicMock()
 
-    metadata = {"python_version": "3.11.3", "sdk_version": "2.3.3"}
-    gcc.register_function(funk, metadata=metadata)
+    metadata = {
+        "python_version": "{}.{}.{}".format(*sys.version_info),
```

Comment:
This is a test, so manually creating it ie. "python_version": "3.99.1" would be a better test to ensure it's not coincidentally filled in by something else?

Reply:
nevermind - it is testing that the default python_version is the result, so not hardcoded version.